atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1125,7 +1130,7 @@ void WebContents::LoadURL(const GURL& url, const mate::Dictionary& options) {
 
   std::string user_agent;
   if (options.Get("userAgent", &user_agent))
-    web_contents()->SetUserAgentOverride(user_agent);
+    web_contents()->SetUserAgentOverride(user_agent, false);
```

Comment:
Does this `false` mean that we won't override the UA after a navigation?

Reply:
@MarshallOfSound ?

Reply:
I don't think so 🤔 The change that created this new option implied that `false` was equivalent to the old behavior.

https://chromium-review.googlesource.com/c/chromium/src/+/846713

Reply:
The false value defines whether we should propagate UA override values to renderer initiated new window navigations when there is no navigation entry, it doesn't disable overriding UA after a navigation. Also this propagation of UA doesn't help us, since the new window doesn't share the same webContents. The change doesn't regress any existing behavior.